packages/cli/test/unit/update-index.unit.js

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+const updateIndex = require('../../lib/update-index');
+const assert = require('yeoman-assert');
+const path = require('path');
+const util = require('util');
+const fs = require('fs');
+const writeFileAsync = util.promisify(fs.writeFile);
+
+const testlab = require('@loopback/testlab');
+const expect = testlab.expect;
+const TestSandbox = testlab.TestSandbox;
+
+// Test Sandbox
+const SANDBOX_PATH = path.resolve(__dirname, '.sandbox');
+const sandbox = new TestSandbox(SANDBOX_PATH);
+const expectedFile = path.join(SANDBOX_PATH, 'index.ts');
+
+describe('update-index unit tests', () => {
+  beforeEach('reset sandbox', () => sandbox.reset());
```

Comment:
Is it possible to just provide `sandbox.reset` since it isn't async anymore?

Reply:
I've tried and I get the following error message if I do `beforeEach(sandbox.reset)`:

```
1) update-index unit tests
       "before each" hook: reset for "creates index.ts when not present":
     TypeError: this.validateInst is not a function
      at Context.reset (packages/testlab/src/test-sandbox.ts:31:27)
```

Reply:
Looks like it has to do with scoping :(.
On the other hand, do you think `async () => await sandbox.reset()` is cleaner than `() => sandbox.reset())`?

Reply:
I like the current format -- it's how we use `sandbox.reset` in the beforeEach hooks of other tests. Consistency is good IMO